docs/subnets/add-subnet.md

```diff
@@ -16,8 +16,15 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
```

Comment:
@dudunog could you please update the PR description linking the PR with the issue(s) it's closing?
For details, see [GitHub syntax for linking issues to PRs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword#linking-a-pull-request-to-an-issue-using-a-keyword)